core/services/bff/tsconfig.json

```diff
@@ -0,0 +1,39 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "strict": false,
```

Comment:
We should use strict mode as a good practice.

Reply:
That's why there's a big [TEMP] on this commit :sweat_smile: 

I will remove it when finishing the Wifi service componentization.